plenario/api/point.py

```diff
@@ -110,9 +69,9 @@ def detail():
 def grid():
     fields = ('dataset_name', 'resolution', 'buffer', 'obs_date__le', 'obs_date__ge',
               'location_geom__within')
-    validated_args = validate(DatasetRequiredValidator(), request.args.to_dict())
+    validated_args = validate(DatasetRequiredValidator(only=fields), request.args.to_dict())
```

Comment:
/grid was not restricted to just checking fields, and would check any field listed in the DatasetRequiredValidator